src/cowSdk.ts

```diff
@@ -0,0 +1,44 @@
+import { OrderBookApi } from '@cowprotocol/cow-sdk/order-book'
+import { SupportedChainId } from '@cowprotocol/cow-sdk'
+import { SubgraphApi } from '@cowprotocol/cow-sdk/subgraph'
+import { MetadataApi } from '@cowprotocol/cow-sdk/metadata'
+
+const orderBookSDKCache: Record<SupportedChainId, { prod: OrderBookApi | null; staging: OrderBookApi | null }> = {
+  [SupportedChainId.MAINNET]: { prod: null, staging: null },
+  [SupportedChainId.GNOSIS_CHAIN]: { prod: null, staging: null },
+  [SupportedChainId.GOERLI]: { prod: null, staging: null },
+}
+
+export function orderBookSDK(chainId: SupportedChainId, env: 'prod' | 'staging' = 'prod'): OrderBookApi {
+  const cached = orderBookSDKCache[chainId][env]
+
+  if (!cached) {
+    const sdk = new OrderBookApi(chainId, env)
```

Comment:
I should comment this on the SDK PR, but here I can see it being used so I have a better grasp of the changes.

We intentionally moved away from having one api instance per chain and moved the config as part of each individual call

We should discuss this in the sync 

cc @anxolin

Reply:
It's a good point, I will refactor it